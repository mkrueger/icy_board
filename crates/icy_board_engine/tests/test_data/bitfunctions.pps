INTEGER A
A = 0
BitSet A, 4
PRINTLN A, IsBitSet(A, 4), IsBitSet(A, 1)
BitSet A, 1
PRINTLN A, IsBitSet(A, 4), IsBitSet(A, 1)
BitClear A, 4
PRINTLN A, IsBitSet(A, 4), IsBitSet(A, 1)
BitClear A, 1
PRINTLN A, IsBitSet(A, 4), IsBitSet(A, 1)
