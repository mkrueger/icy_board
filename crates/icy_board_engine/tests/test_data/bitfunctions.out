1610
1811
201
000
